admin/server/projects.go

```diff
@@ -85,7 +85,7 @@ func (s *Server) ListProjectsForOrganization(ctx context.Context, req *adminv1.L
 
 	dtos := make([]*adminv1.Project, len(projs))
 	for i, p := range projs {
-		dtos[i] = s.projToDTO(p, org.Name)
+		dtos[i] = s.projToDTO(ctx, p, org.Name)
 	}
```

Comment:
It's not really safe to do this in a loop, can lead to a lot of additional lookups (N+1 query problem)